cmd/fyne/internal/commands/build.go

```diff
@@ -443,9 +443,5 @@ func normaliseVersion(str string) string {
 	if pos := strings.Index(str, "-0.20"); pos != -1 {
 		str = str[:pos] + "-dev"
 	}
-
```

Comment:
This, and a couple of other things, have been added to the tools repo only and should not be reverted